src/main/scala/nl/knaw/dans/bag/Deposit.scala

```diff
@@ -351,22 +358,50 @@ object Deposit {
 
   val depositPropertiesName = "deposit.properties"
 
-  def empty(baseDir: File,
-            algorithms: Set[ChecksumAlgorithm] = Set(ChecksumAlgorithm.SHA1),
-            bagInfo: Map[String, Seq[String]] = Map.empty,
-            state: State,
-            depositor: Depositor,
-            bagStore: BagStore): Try[Deposit] = {
+  /**
+   * Creates a new `nl.knaw.dans.bag.Deposit` with an empty `nl.knaw.dans.bag.DansBag` inside.
+   *
+   * @param baseDir the directory for the new Deposit
+   * @param algorithms the algorithms with which the checksums for the (payload/tag) files are calculated.
+   *                   If none are provided, SHA1 is used.
+   * @param bagInfo the entries to be added to `bag-info.txt`
+   * @param state the state to be set in the `deposit.properties`' state.label
+   * @param depositor the depositor to be set in the `deposit.properties`' `depositor.userId`
+   * @param bagStore the bagId for the `bag-dir` in this `nl.knaw.dans.bag.Deposit`
+   * @return if successful, returns a `nl.knaw.dans.bag.Deposit` object representing the deposit located at `baseDir`
+   *         else returns an exception
+   */
+  def from(baseDir: File,
+           algorithms: Set[ChecksumAlgorithm] = Set(ChecksumAlgorithm.SHA1),
+           bagInfo: Map[String, Seq[String]] = Map.empty,
+           state: State,
+           depositor: Depositor,
+           bagStore: BagStore): Try[Deposit] = {
     if (baseDir.exists)
       Failure(new FileAlreadyExistsException(baseDir.toString))
     else
       for {
         bag <- DansBag.empty(baseDir / bagStore.bagId.toString, algorithms, bagInfo)
-        properties = DepositProperties.empty(state, depositor, bagStore)
+        properties = DepositProperties.from(state, depositor, bagStore)
         _ <- properties.save(depositProperties(baseDir))
       } yield new Deposit(baseDir, bag, properties)
   }
 
+  //TODO wouldn't it be better to be able to provide the payloadDir and a baseDir for the Deposit separately?
+  /**
+   * Creates a new Deposit, as a parent-directory to the `payloadDir`, moving the files in the
+   * `payloadDir` to a Bag in the Deposit.
+   *
+   * @param payloadDir the directory containing the payload files for the bag. Here the Deposit will
```

Comment:
Just the uploaded files. i.e. the `data` folder in the DansV0Bag, or also files like datasetmetadata and files.xml? This is still confusing for me.